@here/harp-mapview/lib/DataSource.ts

```diff
@@ -151,6 +151,10 @@ export abstract class DataSource extends THREE.EventDispatcher {
         // to be overloaded by subclasses
     }
 
+    clearCache() {
```

Comment:
Please add comments describing the purpose of the method

Reply:
Done. But it should be obvious from the method name.

Reply:
True but this is public API in a core base class that will be implemented by users. For these classes we should have documentation, even if it is trivial.